theano/sandbox/cuda/blas.py

```diff
@@ -14,6 +14,220 @@
                                            gpu_contiguous)
 from theano.tensor import as_tensor_variable
 
+class BatchedDotOp(GpuOp):
+
+    def __str__(self):
+        return 'BatchedDotOp'
+
+    def __eq__(self, other):
+        return type(self) == type(other)
+
+    def __hash__(self):
```

Comment:
Remove those 3 methods. This is the old way of doing things. Add this attribute:

`__props__ = ()`